support-frontend/assets/pages/weekly-subscription-checkout/helpers/deliveryDays.js

```diff
@@ -14,7 +14,8 @@ const getWeeklyDays = (today: ?number): Date[] => {
   const now = new Date(today || new Date().getTime());
   const currentWeekday = now.getDay();
   const weeksToAdd =
-      currentWeekday > extraDelayCutoffWeekday
+      currentWeekday > extraDelayCutoffWeekday || 
+      currentWeekday === 0 // Sunday is considered the last day of the week
```

Comment:
I know we discussed this a while ago, but  I still think we should align whatever the fulfilment cutoff is with the delivery cutoff week commencing, and then allow all valid days after that week.